test/tests/userobjects/particle_initializer/per_element/errors/tests

```diff
@@ -27,14 +27,6 @@
       expect_err = 'Range check failed for parameter UserObjects/initializer/mass'
       detail = 'tries to give particles a negative mass'
     []
-    [too_few_distributions]
-      type = 'RunException'
-      input = 'errors.i'
-      allow_test_objects = true
-      cli_args = "UserObjects/initializer/velocity_distributions='zero zero'"
-      expect_err = 'You must provide 3 distributions, one for each velocity component.'
-      detail = 'does not provide enough distributions to sample for velocity initialization.'
-    []
```

Comment:
Why not update this to provide too few velocities and ensure it does indeed create enough (and maybe creates a warning to ensure the user is aware of that).